spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -54,8 +55,10 @@
 from spyder.utils.misc import getcwd_or_home
 from spyder.utils.qthelpers import (add_actions, create_action,
                                     mimedata2url)
+from spyder.utils.stringmatching import get_search_scores
 from spyder.plugins.variableexplorer.widgets.importwizard import ImportWizard
 from spyder.plugins.variableexplorer.widgets.texteditor import TextEditor
+from spyder.preferences.shortcuts import CustomSortFilterProxy

```

Comment:
Same here (as above)?

Reply:
I think here the import order is fine, `plugins` takes precedence of `preferences`, right?

Reply:
Opps yes!